src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -160,6 +161,9 @@ type MoneyRequestConfirmationListProps = MoneyRequestConfirmationListOnyxProps &
     /** Whether we're editing a split expense */
     isEditingSplitBill?: boolean;
 
+    /** Whether we're shwowing a created split expense */
+    isSplitBillDetailsPage?: boolean;
```

Comment:
I think we can make the naming more useable here, for example, `shouldReceiptPressable`.

Reply:
`canDIsplayReceipt` sounds good to me.